modules/http-api/src/test/java/org/mule/service/http/api/domain/ParameterMapTestCase.java

```diff
@@ -15,17 +15,12 @@
 import static org.hamcrest.core.IsNull.nullValue;
 import static org.junit.Assert.assertThat;
 
-import org.mule.service.http.api.domain.ParameterMap;
-import org.mule.tck.junit4.AbstractMuleTestCase;
-import org.mule.tck.size.SmallTest;
-
 import java.util.Collection;
 import java.util.HashMap;
 
 import org.junit.Test;
 
-@SmallTest
-public class ParameterMapTestCase extends AbstractMuleTestCase {
```

Comment:
It was not really needed since it's just a simple test. Besides I don't have access to mule testing framework from the HTTP API.